apps/examples/wifi_controller/Makefile

```diff
@@ -22,17 +22,17 @@ include $(APPDIR)/Make.defs
 
 # built-in application info
 
-APPNAME = wm_test
+APPNAME = wictl
```

Comment:
Is it ok to change a command name on TASH???

Reply:
Because wm_test is just a simple app, which is defined in apps/examples/wifi_manager_sample/, we think it has no problem to change the name of it (even in tash command). Please let me know if I need to know anything about changing the tash command in this case.